device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/common/Constants.kt

```diff
@@ -0,0 +1,38 @@
+package com.builttoroam.devicecalendar.common
+
+import android.provider.CalendarContract
+
+class Constants {
```

Comment:
i think this should be public final class according to this article i saw before
https://blog.egorand.me/where-do-i-put-my-constants-in-kotlin/

Reply:
All classes of Kotlin are final by default

Reply:
oh actually i misread, it's the generated Java code. Seems the recommendation is to have a top level/global const but can look into that later